Not able to create custom binaries

# Github issue #
go build is not creating binaries.

## Issue type ##

- Bug Report

## Build number #

commit bf1a76baed5abbcea0d3f9249916a8b876cb548d (HEAD -> master, origin/master, origin/HEAD)
Author: Cristian Măgherușan-Stanciu @magheru_san <cristim@users.noreply.github.com>
Date:   Sun May 27 12:05:37 2018 +0200

    Update README.md


```
I am trying to create a custom binary by following https://github.com/cristim/autospotting/blob/master/CUSTOM_BUILDS.md

In the last step, I do go build, but there is no build generated.

$ make full-test
error	some files did not pass go fmt, fix the following formatting diff:
make: *** [fmt-check] Error 1

Is there a step that I am missing. 
```

## Configuration ##

Mac OS

## Environment ##

```
$ go version
go version go1.10.1 darwin/amd64
$ git version
git version 2.15.0
$ aws --version
aws-cli/1.15.4 Python/3.5.0 Darwin/16.7.0 botocore/1.10.4
```

## Summary ##

Running go build on ~/godev/src/github.com/cristim/autospotting does not create binaries as intended.
$ make full-test
error	some files did not pass go fmt, fix the following formatting diff:
make: *** [fmt-check] Error 1


## Steps to reproduce ##

Followed the steps given in https://github.com/cristim/autospotting/blob/master/CUSTOM_BUILDS.md
to compile binaries locally.

<!-- You can also paste gist.github.com links for larger content -->

## Expected results ##

build folder with the binary will be created

## Actual results ##

no build folder present

--- a/Makefile
+++ b/Makefile
@@ -61,7 +61,7 @@ upload: archive                                              ## Upload binary
 .PHONY: upload
 
 vet-check:                                                   ## Verify vet compliance
-ifeq ($(shell go tool vet -all -shadow=true $(GOFILES) 2>&1 | wc -l), 0)
+ifeq ($(shell go tool vet -all -shadow=true $(GOFILES) 2>&1 | wc -l | tr -d '[:space:]'), 0)
 	@printf "ok\tall files passed go vet\n"
 else
 	@printf "error\tsome files did not pass go vet\n"
@@ -70,12 +70,12 @@ endif
 .PHONY: vet-check
 
 fmt-check:                                                   ## Verify fmt compliance
-ifneq ($(shell gofmt -l -s $(GOFILES) | wc -l), 0)
+ifeq ($(shell gofmt -l -s $(GOFILES) | wc -l | tr -d '[:space:]'), 0)
+	@printf "ok\tall files passed go fmt\n"
+else
 	@printf "error\tsome files did not pass go fmt, fix the following formatting diff:\n"
 	@gofmt -l -s -d $(GOFILES)
 	@exit 1
-else
-	@printf "ok\tall files passed go fmt\n"
 endif
 .PHONY: fmt-check
 

